packages/ui-checkbox/src/CheckboxGroup/index.tsx

```diff
@@ -111,40 +109,46 @@ class CheckboxGroup extends Component<CheckboxGroupProps> {
   get value() {
     if (
       typeof this.props.value === 'undefined' &&
-      // @ts-expect-error ts-migrate(2339) FIXME: Property 'value' does not exist on type 'Readonly<... Remove this comment to see the full error message
       typeof this.state.value === 'undefined'
     ) {
       return []
     } else {
       return typeof this.props.value === 'undefined'
-        ? // @ts-expect-error ts-migrate(2339) FIXME: Property 'value' does not exist on type 'Readonly<... Remove this comment to see the full error message
-          [...this.state.value]
+        ? [...this.state.value]
         : [...this.props.value]
     }
   }
 
   renderChildren() {
     const { children, name, size, disabled, readOnly } = this.props
 
-    // @ts-expect-error ts-migrate(6133) FIXME: 'index' is declared but its value is never read.
-    return Children.map(children, (child, index) => {
+    return Children.map(children, (child) => {
```

Comment:
Just FYI: In my PR (https://github.com/instructure/instructure-ui/pull/881) I made a lot of changes in these 2 utils and had to refactor the codes in a lot of components. Depending which PR is merged first, we gonna have to resolve the conflicts here.